### PR TITLE
Port to AIX platform

### PR DESCRIPTION
**Pull request by V-for-Vasili**
Allows compiling and using ninja with gcc on AIX. (corrected PR)

### CODE REVIEW

File: src/aix_port.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2012 Google Inc. All Rights Reserved.
```
Comment (jonesmz) [inline src/aix_port.h]: Is this accurate?

**Comment by jonesmz** [comment]
Maybe you can squash all 3 commits into a single one?

**Comment by V-for-Vasili** [comment]
Done

**Comment by V-for-Vasili** [comment]
@jonesmz  Done.

**Pull request merged**